casepro/pods/views.py

```diff
@@ -33,4 +37,15 @@ def perform_pod_action(request, pk):
         return JsonResponse(
             {'reason': 'JSON decode error', 'details': e.message}, status=400)
 
-    return JsonResponse(pod.perform_action(data))
+    case_id = data.get('case_id')
+    if case_id is None:
+        return JsonResponse(
+            {'reason': 'Request object needs to have a "case" field'}, status=400)
```

Comment:
Yes, I forgot to change the error message when I changed the field above, thanks.

Resolved in https://github.com/praekelt/casepro/pull/51/commits/5ab83af30e128619399b2f7917265d23c1565b1d